src/Components/Facility/Consultations/DailyRoundsList.tsx

```diff
@@ -19,19 +21,26 @@ interface Props {
 export default function DailyRoundsList({ consultation }: Props) {
   const [consultationId] = useSlugs("consultation");
   const { t } = useTranslation();
+  const [query, setQuery] = useState<QueryParams>();
 
   const consultationUrl = `/facility/${consultation.facility}/patient/${consultation.patient}/consultation/${consultation.id}`;
 
   return (
     <PaginatedList
       route={routes.getDailyReports}
       pathParams={{ consultationId }}
+      query={query}
     >
       {({ refetch }) => (
         <>
           <div className="flex flex-1 justify-between">
             <PageTitle title="Update Log" hideBack breadcrumbs={false} />
-            <DailyRoundsFilter onApply={(query) => refetch({ query })} />
+            <DailyRoundsFilter
+              onApply={(query) => {
+                setQuery(query);
+                refetch();
```

Comment:
But does `refetch` needs to be called? Since `query` is now an external state, and is being passed to useQuery, it'll auto refresh whenever the external state changes right?

cc: @Ashesh3

Reply:
```suggestion
```

Reply:
Correct 😅 Fixed.